XenModel/Messages.Designer.cs

```diff
@@ -6063,7 +6063,7 @@ public class Messages {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Could not find the bond master in [XenCenter]&apos;s cache..
+        ///   Looks up a localized string similar to Could not find the bond interface in [XenCenter]&apos;s cache..

```

Comment:
should this be interface or coordinator?

Reply:
Master in the context of bond/interfaces has been renamed to interface. The term coordinator is used for pools.